src/test/java/reposense/parser/RepoConfigurationTest.java

```diff
@@ -159,6 +161,24 @@ public void repoConfig_ignoresStandaloneConfig_success()
         TestUtil.compareRepoConfig(expectedConfig, actualConfig);
     }
 
+    @Test
+    public void repoConfig_ignoresStandaloneConfigInCli_success()
+            throws ParseException, GitDownloaderException, IOException {
```

Comment:
Seems `IOException` is not thrown here

Reply:
'RepoConfigCsvParser.parse()' method throws this exception